HPXMLtoOpenStudio/resources/utility_bills.rb

```diff
@@ -1,81 +1,23 @@
 # frozen_string_literal: true
 
 class UtilityBills
-  def self.get_rates_from_eia_data(runner, state_code, fuel_type, fixed_charge, marginal_rate = nil)
-    if state_code == 'US'
-      if fuel_type == HPXML::FuelTypeElectricity
-        state_name = 'United States'
-      else
-        state_name = 'U.S.'
-      end
-    else
-      state_name = Constants.StateCodesMap[state_code]
-    end
-    return if state_name.nil?
+  def self.get_rates_from_eia_data(_runner, state_code, fuel_type, fixed_charge, marginal_rate = nil)
+    msn_codes = Constants.StateCodesMap.map { |x| x[0] }.uniq
```

Comment:
I'm not sure why you are ensuring a unique set (isn't it already unique?), but it seems like you just want the hash keys? Untested suggestion:

```suggestion
    msn_codes = Constants.StateCodesMap.keys
```